third_party/amd/lib/TritonAMDGPUToLLVM/LoadStoreOpToLLVM.cpp

```diff
@@ -1548,14 +1548,40 @@ struct AtomicRMWOpConversion
     rmwPtr = b.ptrtoint(i64_ty, rmwPtr);
 
     auto *curBlock = rewriter.getInsertionBlock();
-    auto *afterLoopBlock = curBlock->splitBlock(rewriter.getInsertionPoint());
+    auto *leaderBlock = curBlock->splitBlock(rewriter.getInsertionPoint());
```

Comment:
I feel `leaderBlock` is not a proper name anymore given now we can have multiple threads doing the atomic. What about naming it `atomicBlock`.

Reply:
Fixed.